src/reducers/dashboardFeed.js

```diff
@@ -16,6 +23,7 @@ export default function dashboardFeed(state = initialState, action) {
         activeThread: action.threadId,
       });
     case 'SELECT_FEED_COMMUNITY':
+      storeItem(LAST_ACTIVE_COMMUNITY_KEY, action.communityId);
```

Comment:
Reducers should be pure—let's move that to happen in the `selectFeedCommunity` action instead!